Src/BootCamp.Chapter/Calculate.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Calculate
+    {
+        private ILogger _logger;
+
+        public Calculate()
+        {
+            _logger = InitialiseLogger();
```

Comment:
Wrong. You are missing a point, you should be using the implementation of logger across the whole application consistently, now you just hardcoded it.

Reply:
Agreed, the entire application should utilise the same 'logger' everything the application to run. Initialisation of logger is now generic and instantiated within the Program.cs file.